crates/egui/src/data/input.rs

```diff
@@ -563,6 +635,129 @@ pub enum Key {
     F20,
 }
 
+impl Key {
+    /// Emoji or name representing the key
+    pub fn symbol_or_name(self) -> &'static str {
+        match self {
+            Key::ArrowDown => "⏷",
+            Key::ArrowLeft => "⏴",
+            Key::ArrowRight => "⏵",
+            Key::ArrowUp => "⏶",
+            _ => self.name(),
```

Comment:
https://wincent.com/wiki/Unicode_representations_of_modifier_keys has even more Unicode symbols for special keys

Reply:
Nice link I think I'll add it as a TODO, because we need to check what characters are supported by a font before we use them (the egui default font supports those arrow triangles, but no others in that link).